draft-retana-idr-bgp-quic-02.xml

```diff
@@ -130,7 +130,7 @@
        control of the connection.
      </t>
      <t>
-        This document specifies the procedures for BGP to use QUIC as a transport protocol with a mechanism to carry Network Layer protocols (AFI/SAFI) over individual streams.  The Network layer protocols are identified using a combination of Address Family (AF) and Subsequent Address Family (SAFI), as described in <xref target="RFC4760"/>. These per-AFI/SAFI streams (function channels) and the associated control mechanism (control channel) for the session are called "BGP channels". In one BGP over QUIC (BoQ) connection, one control channel and one or more function channels are used to carry routing information.
```

Comment:
We'll be wanting to keep AFI here from 4760.
The bare "address family" is regularly used in bgp-land to imply the afi/safi pairing, but isn't terribly well supported by the core documents.

Reply:
I'm not sure what you mean, because I replaced "AF" with "AFI" in my commit.  If you want me to undo the replacement, OK, but it seems awkward to me to use "AF" and "SAFI" in that sentence.

Reply:
Insufficient coffee first thing in the morning error.  This is correct.  Thanks.